### ISSUE TITLE
Selecting backlinks results in an invalid syntax (EdgeQLSyntaxError: Unexpected '<')

### ISSUE THREAD

**Issue by OriginalEXE** [edgedb/edgedb-js#329]
- EdgeDB Version: `0.20.7`
- EdgeDB CLI Version: `1.1.1+932d5c6`

When trying to select backlinks on a model, the generated query looks something like this:
```
SELECT __some_scope {
  id,
  name,
  multi <some_model[is some_other_model] := (
...
```

EdgeDB complains that `<` there is unexpected, and I see no way to alias it in the query builder to something else. Replacing  `multi <some_model[is some_other_model] :=` with something like `some_other_models :=` makes the query work.

**Comment by OriginalEXE** [comment]
Thanks for fixing the issue and sharing the way to alias this, I don't know why I failed to figure that syntax out :)

**Comment by colinhacks** [comment]
A fix has landed in `v0.20.9` @OriginalEXE

**Comment by OriginalEXE** [comment]
Awesome, thank you for your work, people, and for such a nice product, really having fun with EdgeDB :)
